### DISCUSSION TITLE
Some more issues with XHTML, I have a fix in a local branch if interested

### DISCUSSION THREAD

**Discussion by ljungbaeck** [strictdoc-project/reqif#95] [Q&A] [answered]
Hi Stanislav,

I've finally come around to install github and git on my machine so I could help out with publishing some solutions to problems I've found on my end.

Using my full Doors-example, I had some xhtml values that didn't got the correct values, example below:

<THE-VALUE><reqif-xhtml:div>some data<reqif-xhtml:br />&#9;Subtitle<reqif-xhtml:br />some data<reqif-xhtml:br />&#9;More data<reqif-xhtml:br />&#9;&#9;using even more data</reqif-xhtml:div></THE-VALUE>

Also, during the unparsing of my examples, I got an error of a missing template for the type BOOLEAN so I've added that as well.

//

**Comment by stanislaw** [comment]
Hi @ljungbaeck , thanks for coming back. I don't see your branch. Have you made the repo public or pushed it?

**Reply by ljungbaeck** [reply]
![image](https://user-images.githubusercontent.com/75385123/218124412-441fd3f2-7408-472f-bb24-f553a85dc5ad.png)

Not able to push it to the current project.. I can create a fork though if you prefer that?

**Reply by stanislaw** [reply]
Yes, please create a fork. That's a common practice.

**Comment by stanislaw** [comment]
Is there a chance I could see the anonymized ReqIf file you are working with? From your branch, I cannot see which problem you are solving and we anyway need a good ReqIF example to write a test case.

Please `pip install reqif -U` and try the `reqif anonymize` command. When everything works, please send me the file via an email, and I will integrate it to the repository.

**Reply by stanislaw** [reply]
Or if you are working on the repository anyway, fetch the latest changes from the main branch and run the anonymize command as
`python reqif/cli/main.py anonymize ...`.

**Comment by stanislaw** [comment]
In fact, I don't need the whole ReqIF you are working with but only the reduced examples for each of the aspects. Example: https://github.com/strictdoc-project/reqif/blob/main/tests/integration/reqif/SPEC-OBJECTS/01_basic_spec_object/sample.reqif.

It looks like you are dealing with two issues:

XHTML and BOOLEAN

so ideally I would get two reduced .reqif files from you that I could work against.

**Comment by ljungbaeck** [comment]
I tried to run the anonymize using my full reqif-file but it failed (I'll try to narrow down on why). So instead I'm sending you a small snippet containing more complex xhtml values as well as a boolean which is failing if I try to parse the file and then unparse it directly. Sending it through separate e-mail.

//

**Comment by stanislaw** [comment]
Hey @ljungbaeck, it would be great to know if the latest version of reqif is working well for you now. Thanks.

**Reply by ljungbaeck** [reply]
Hey, I've tried the latest version and it seems to parse and unparse without any errors. When it comes to xhtml, the values aren't stripped of the "<reqif-xhtml:" occurances, but maybe that's by design. If so I will strip all that on my end! The anonymize command did work as well! 

Big thanks for the implementation!

**Reply by stanislaw** [reply]
Glad to hear! The <reqif-xhtml> is currently by design but we can easily consider automating that as well. Please open a separate ticket for that.

**Comment by stanislaw** [comment, marked as answer]
I am closing this ticket because the missing feature have been implemented.
